src/tests/Microsoft.PowerFx.Connectors.Tests/PowerPlatformConnectorTests.cs

```diff
@@ -242,11 +246,22 @@ public async Task AzureBlobConnector_UploadFile(bool useSwaggerParameter, bool u
 
             // Now execute it...
             var engine = new RecalcEngine(config);
-            RuntimeConfig runtimeConfig = new RuntimeConfig().AddRuntimeContext(new TestConnectorRuntimeContext("AzureBlobStorage", client));
+            var runtimeContext = new TestConnectorRuntimeContext("AzureBlobStorage", client);
+            RuntimeConfig runtimeConfig = new RuntimeConfig().AddRuntimeContext(runtimeContext);
             testConnector.SetResponseFromFile(@"Responses\AzureBlobStorage_Response.json");
 
-            var result = await engine.EvalAsync(@"AzureBlobStorage.CreateFile(""container"", ""bora4.txt"", ""abc"").Size", CancellationToken.None, options: new ParserOptions() { AllowsSideEffects = true }, runtimeConfig: runtimeConfig).ConfigureAwait(false);
+            // 3rd parameter here expects a blob.
+            CheckResult check = engine.Check(@"AzureBlobStorage.CreateFile(""container"", ""bora4.txt"", AsBlob(""abc"")).Size", new ParserOptions() { AllowsSideEffects = true });

```

Comment:
>AsBlob(""abc"")) [](http://example.com/codeflow?start=105&length=16)

No coercion in place, we create a blob with a function and pass the blob to CreateFile